src/components/Chloropleth.jsx

```diff
@@ -314,7 +314,7 @@ const Chloropleth = (props) => {
             onHover={e => {
               const [feature] = e.features
               if (feature && 'value' in feature.properties) {
-                if (feature.properties.latitude === undefined) {
+                if (feature.properties.lat === undefined) {
```

Comment:
do you prefer the abbreviation? Think I ended up with the full name in the query params